userdata/Manifest/releng-secrets.json

```diff
@@ -1,12 +1,4 @@
 [
-  "crash-stats-api.token",
-  "gapi.data",
-  "google-oauth-api.key",
-  "mozilla-api.key",
-  "mozilla-desktop-geoloc-api.key",
-  "mozilla-fennec-geoloc-api.key",
   "relengapi.tok",
-  "tc-sccache.boto",
```

Comment:
I suspect that these are still in use for sccache on windows.

Reply:
I can readd. [Searchfox doesn't show any usage](https://searchfox.org/mozilla-central/search?q=tc-sccache.boto&path=) though ?

Reply:
i will remove the sccache secret as well. it isn't used because we have an iam role on the builders that gives access to the sccache buckets (eg: https://github.com/mozilla-releng/OpenCloudConfig/blob/19d4f80/userdata/Manifest/gecko-1-b-win2012.json#L1461).